css/style.css

```diff
@@ -21,7 +21,6 @@ h1 {
 	text-align: center;
 	font-size: 1.85em;
 	font-family: "REM", sans-serif;
-	font-optical-sizing: auto;
```

Comment:
This was removed because of the css validator, it also got me to remove 'float: center'